internal/config/options.go

```diff
@@ -188,148 +187,68 @@ var defaultOptions = Options{
 	GRPCAddr:                ":443",
 	GRPCClientTimeout:       10 * time.Second, // Try to withstand transient service failures for a single request
 	GRPCClientDNSRoundRobin: true,
+	viper:                   viper.New(),
 }
 
-// NewOptions creates a new Options struct with only viper initialized
-func NewOptions() *Options {
-	o := Options{}
-	o.viper = viper.New()
-	return &o
+// NewDefaultOptions returns a new set of default options. It's the caller's
+// responsibility to do a follow up Validate call.
+func NewDefaultOptions() Options {
+	return defaultOptions
```

Comment:
This should be a reference, shouldn't it?

Reply:
Updated.